types/AbstractBaseType.php

```diff
@@ -451,6 +451,17 @@ public function rawValue($value) {
     }
 
     /**
+     * This is called when a single string is needed to represent this Type's current
+     * value as a single (non-HTML) string. Eg. in a dropdown or in autocompletion.
+     *
+     * @param string $value
+     * @return string
+     */
+    public function displayValue($value) {
```

Comment:
I'm curious about the naming. Whenever there is a difference between `rawValue` and "non-raw-value", isn't the raw value the one we do _not_ want to display?